### DISCUSSION TITLE
Read timed out.

### DISCUSSION THREAD

**Discussion by 13703455592** [jertel/elastalert2#845] [Q&A] [answered]
elastalert.util.EAException: Error loading file /opt/elastalert2/examples/rules/test.yaml: Error initializing rule power: Error searching for existing terms: ConnectionTimeout('TIMEOUT', "HTTPConnectionPool(host='10.5.5.141', port=9200): Read timed out. (read timeout=20)", ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='10.5.5.141', port=9200): Read timed out. (read timeout=20)")))
该错误是因为索引过大嘛，需要读取的索引大小为400gb

**Comment by jertel** [comment, marked as answer]
Try increasing the `es_conn_timeout` value in the config yaml. See the documentation: https://elastalert2.readthedocs.io/en/latest/elastalert.html?highlight=es_conn_timeout
